src/SFA.DAS.ProviderRequestApprenticeTraining.Infrastructure/Services/SessionStorage/ProviderResponse.cs

```diff
@@ -10,5 +10,6 @@ public class ProviderResponse
         public List<Guid> SelectedEmployerRequests { get; set; } = new List<Guid>();
         public string SelectedEmail { get; set; }
         public bool HasSingleEmail { get; set; }
+        public string SelectedPhoneNumber { get; set; }
```

Comment:
I think you'll need HasSinglePhoneNumber for enabling the change link on the CheckAnswers - but maybe in that story you will add it

Reply:
yes, its in 2052